ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -146,7 +148,7 @@ run tracers chainDbTracer diffusionTracers diffusionArguments networkMagic
                            ()
           networkApps = consensusNetworkApps
             nodeKernel
-            nullProtocolTracers
+            protocolTracers
```

Comment:
Huh, I'm surprised to see this. I remember seeing flags in the node to trace the different protocols, apparently they didn't work?

Reply:
Yeah so the protocol tracers were never being passed through.